### PR TITLE
Update View value_type and const_value_type for compile time checks

### PR DESCRIPTION
**Pull request by ndellingwood**
More fixes related to kokkos/kokkos-kernels#1367

### CODE REVIEW

File: src/sparse/impl/KokkosSparse_getDiagCopyWithOffsets_impl.hpp [hunk 1]
```diff
@@ -88,7 +88,8 @@ struct CrsMatrixGetDiagCopyWithOffsetsFunctor {
         static_cast<int>(DiagType::rank) == 1,
         "The DiagType template parameter must be a 1-D Kokkos::View.");
     static_assert(
-        std::is_same<DiagType, typename DiagType::non_const_type>::value,
+        std::is_same<typename DiagType::value_type,
+                     typename DiagType::non_const_value_type>::value,
```
Comment (dalg24) [inline src/sparse/impl/KokkosSparse_getDiagCopyWithOffsets_impl.hpp:92]: or `!std::is_const<typename DiagType::value_type>::value`

**Review by dalg24 — APPROVED** [review]

**Review by e10harvey — APPROVED** [review]

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9_GCC720_Light_Tpls_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 183</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10_Tpls_CUDA10_LayoutRight</h3>
<ul>
  <li>Build Num: 176</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 986</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 630</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 974</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 961</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 366</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: KOKKOSKERNELS (ndellingwood/kokkos-kernels)</dt>
  <ul>
    <li>Branch: issue-1367</li>
    <li>SHA: ffaa347749f88ac37eb89d35c8a531d60dda90c8</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: ndellingwood
</details>

**Comment by ndellingwood** [comment]
Failing jobs seem to be Jenkins-related issues?
```
19:09:14   Starting job cuda-9.2.88-Cuda_OpenMP-release
19:09:14 kokkos devices: Cuda,OpenMP
19:09:14 kokkos arch: Power9,Volta70
19:09:14 kokkos options: 
19:09:14 kokkos cuda options: force_uvm,enable_lambda
19:09:14 kokkos cxxflags: -O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wuninitialized 
19:09:14 extra_args: 
19:09:14 kokkoskernels scalars: 'double,complex_double'
19:09:14 kokkoskernels ordinals: int
19:09:14 kokkoskernels offsets: int,size_t
19:09:14 kokkoskernels layouts: LayoutLeft
19:15:22 FATAL: command execution failed
19:15:22 java.io.EOFException
19:15:22 	at java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2831)
19:15:22 	at java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:3306)
19:15:22 	at java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:933)
19:15:22 	at java.io.ObjectInputStream.<init>(ObjectInputStream.java:395)
19:15:22 	at hudson.remoting.ObjectInputStreamEx.<init>(ObjectInputStreamEx.java:49)
19:15:22 	at hudson.remoting.Command.readFrom(Command.java:142)
19:15:22 	at hudson.remoting.Command.readFrom(Command.java:128)
19:15:22 	at hudson.remoting.AbstractSynchronousByteArrayCommandTransport.read(AbstractSynchronousByteArrayCommandTransport.java:35)
19:15:22 	at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:61)
19:15:22 Caused: java.io.IOException: Unexpected termination of the channel
19:15:22 	at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:75)
...
```

Trying a retest

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9_GCC720_Light_Tpls_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 184</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10_Tpls_CUDA10_LayoutRight</h3>
<ul>
  <li>Build Num: 177</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 987</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 631</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 975</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 962</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 367</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue-1367</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>ffaa347749f88ac37eb89d35c8a531d60dda90c8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ca33f614218b5cfc46796fda82ead414bdc4daf0</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;AT: PRE-TEST INSPECTED</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1394</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>
</details>

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Pull Request MUST BE MERGED MANUALLY BY Project Team - This Repo does not support Automerge

**Pull request merged**